packages/client-core/src/admin/common/Location/LocationDrawer.tsx

```diff
@@ -90,15 +91,17 @@ const LocationDrawer = ({ open, mode, selectedLocation, selectedScene, onClose }
 
   const scenes = useFind(assetPath)
   // const locationTypes = useFind(locationTypePath).data
-  const user = useHookstate(getMutableState(AuthState).user)
 
   const locationMutation = useMutation(locationPath)
 
-  const hasWriteAccess = user.scopes.get(NO_PROXY)?.find((item) => item?.type === 'location:write')
   const viewMode = mode === LocationDrawerMode.ViewEdit && !editMode.value
 
   const selectedSceneData = useGet(assetPath, selectedScene!)
 
+  const editorState = getState(EditorState)
```

Comment:
Since this drawer is used in admin panel as well.

Maybe it would be better if you just have a prop in this component as disabledEditButton. And then from studio's `PublishLocation.tsx` you pass that boolean? As I think you are already checking this.

In this control, if value of `disabledEditButton` is undefined then the button should work as it normal works

Reply:
Done! it's now passed as a property